packages/calcite-components/src/components/tab-nav/tab-nav.tsx

```diff
@@ -187,6 +229,7 @@ export class TabNav {
               ref={(el) => (this.activeIndicatorEl = el as HTMLElement)}
             />
           </div>
+          {this.layout === "inline" && this.getOverflowIcons()}
```

Comment:
`getOverflowIcons` has multiple `getBoundingClientRect` calls and is called during render. Can you decouple the dimension calculation from rendering to improve performance?